repository_cli/cli/util.py

```diff
@@ -54,9 +55,14 @@ def get_draft(pid: str, identity: Identity) -> Draft:
     return draft
 
 
-def get_records_service() -> RDMRecordService:
+def get_records_service(data_model="rdm") -> RDMRecordService:
```

Comment:
The return type is not longer correct. Maybe only using the base class RecordService?